opencti-platform/opencti-front/src/private/components/data/import/ImportContent.jsx

```diff
@@ -229,7 +240,7 @@ class ImportContentComponent extends Component {
     if (isCsvConnector && configuration && objectMarking) {
       const parsedConfig = JSON.parse(configuration);
       if (typeof parsedConfig === 'object') {
-        parsedConfig.user_chosen_markings = objectMarking.map((marking) => marking.value);
+        parsedConfig.markings = objectMarking.map((marking) => marking.value);
```

Comment:
why change this property name here ? I think it should stay user_chosen_markings